test/templates/analyze_hadTopTagger_gen_cfg.py

```diff
@@ -160,15 +160,15 @@
 import os
 def getInputFiles(inputFilePath):
     inputFiles = []
-    files_and_subdirectories = os.listdir(inputFilePath)
+    files_and_subdirectories = hdfs.listdir(inputFilePath)
     for file_or_subdirectory in files_and_subdirectories:
         if file_or_subdirectory in zombie_files:
```

Comment:
`os.listdir()` loops over file names in the directory `inputFilePath`, whereas `hdfs.listdir()` loops over full paths of these files. Replace
```python
if file_or_subdirectory in zombie_files:
```
with
```python
if os.path.basename(file_or_subdirectory) in zombie_files:
```